soccer/src/soccer/soccer_mom/soccer_mom_node.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <deque>
+#include <mutex>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <rj_constants/topic_names.hpp>
+// #include <rj_msgs/msg/manipulator_setpoint.hpp>
+// #include <rj_msgs/msg/motion_setpoint.hpp>
+// #include <rj_msgs/msg/team_color.hpp>
+#include <rj_param_utils/param.hpp>
+#include <rj_param_utils/ros2_local_param_provider.hpp>
+
+#include <std_msgs/msg/string.hpp>
+
+#include <rj_msgs/msg/team_color.hpp>
+
+
+
+namespace tutorial {
+
+
+class SoccerMom : public rclcpp::Node {
+public:
+    SoccerMom();
+
+protected:
+
+private:
+
```

Comment:
Make sure to adhere to the doc strings in our Contributions page. These will be enforced in the future.